crates/ingress-rpc/src/service.rs

```diff
@@ -43,6 +72,59 @@ impl<Queue> IngressService<Queue> {
             queue,
         }
     }
+
+    async fn validate_tx(&self, envelope: &OpTxEnvelope) -> Result<()> {
+        let sender = envelope.recover_signer().unwrap_or_default();
+        let account = self.provider.fetch_account_info(sender).await?;
+
+        // skip eip4844 transactions
+        if envelope.is_eip4844() {
+            return Err(anyhow::anyhow!("EIP-4844 transactions are not supported"));
```

Comment:
Are there typed errors for these?

Reply:
there are, but it's spread across different enums like `EthApiError`, `RpcInvalidTransactionError` etc. but will consolidate some using the convenient `RpcInvalidTransactionError::other` available